config/routes.rb

```diff
@@ -19,4 +19,6 @@
 
   resources :activities, only: :create
   resources :jitsi_calls, only: :create
+
+  get '/chat', to: 'home#chat', as: 'chat'
```

Comment:
```suggestion
  get 'chat', to: 'home#chat'
```
I think this should be sufficient but we have to test it first